src/pages/api/projects/index.page.ts

```diff
@@ -0,0 +1,84 @@
+import { queryDBProjects } from 'lib/api/supabase/projects'
+import { DBProjectQueryOpts } from 'models/dbProject'
+import { ProjectTag } from 'models/project-tags'
+import { NextApiHandler } from 'next'
+
+// Searches Juicebox projects matching text query param
```

Comment:
in this api endpoint, query params for defining project sort/filter options are parsed from strings, validated, and formatted as args to the `queryDBProjects()` function